test/generate.js

```diff
@@ -7,7 +7,11 @@ const assert = require('assert').strict;
 const generate = require('../lib/generate');
 
 // Ideally we should find a way to test this without using the network
-describe('generate()', () => {
+// eslint-disable-next-line  func-names, prefer-arrow-callback
```

Comment:
This whole patch should be reverted.

Reply:
I'm trying to figure out why tests were failing on Travis and not in my local env. It looked like it was running into a timeout on the full test suite since the multi-redirect test was taking upwards of 1.5 seconds (local env this takes less than half a second).

This most recent run took a reasonable amount of time, so I'll revert and resubmit.